### PR TITLE
feat: add GIN results for `Peptides-func`

### PR DESCRIPTION
**Pull request by SauravMaheshkar**
This PR aims to add benchmark performance for a simple Graph Isomorphism Network trained on the Peptides-func dataset (5 fold).

The model had the following configuration:

* Number of parameters: 235k
* Batch Size: 128
* Learning Rate: 0.001 (No Scheduler)
* Optimizer: AdamW
* Epochs: 250
* Number of Layers: 5
* Hidden Dimension: 128
* Jumping Knowledge Used
* GIN-ε variant
* No Dropout

The corresponding Weights & Biases run can be found [here](https://wandb.ai/graph-neural-networks/GIN/runs/64k4e1wf?workspace=user-sauravm) and the code used for training the model can be found [here](https://github.com/SauravMaheshkar/Graph-Property-Prediction)

Request for Review: @vijaydwivedi75 @migalkin @rampasek

### CODE REVIEW

**Comment by SauravMaheshkar** [comment]
Closing for now, happy to reopen if there is interest from the maintainers
